app/views/form/page.html.erb

```diff
@@ -75,6 +75,12 @@
         <%= f.hidden_field :check_errors, value: @check_errors %>
       <% end %>
 
+      <% if @page.id == "address" %>
+        <div class="govuk-button-group">
+          <%= govuk_link_to "Clear address and search instead", address_search_input_path(@log.log_type, @log.id), class: "govuk-button govuk-button--secondary" %>
+        </div>
+      <% end %>
```

Comment:
Could this be something similar to guidance_partial? Would probably be safer not to hardcode page IDs